### PR TITLE
OKTA-686715 Clean up figma diagrams

### PR DESCRIPTION
**Pull request by danielmaharry-okta**
## Description:
- **What's changed?** 
- Deleted old flow diagrams already repalce by new Figma diagrams
- Add new wireframe diagram for widget sign in with facebook guide.

- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-686715](https://oktainc.atlassian.net/browse/OKTA-686715)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-case-sign-in-soc-idp/main/aspnet/integrationsteps.md](https://github.com/okta/okta-developer-docs/blob/94fece47452dddf1b1b6d3b553ee5af2cd6e1c87/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Foie-embedded-widget-use-case-sign-in-soc-idp%2Fmain%2Faspnet%2Fintegrationsteps.md) | 66 | 35 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/f0592c4b-e29f-4ed4-8c7a-cfe96aa7bc47) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-case-sign-in-soc-idp/main/go/integrationsteps.md](https://github.com/okta/okta-developer-docs/blob/94fece47452dddf1b1b6d3b553ee5af2cd6e1c87/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Foie-embedded-widget-use-case-sign-in-soc-idp%2Fmain%2Fgo%2Fintegrationsteps.md) | 62 | 37 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/7fd16d39-ee06-482c-82ea-32005d01ce64) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-case-sign-in-soc-idp/main/java/integrationsteps.md](https://github.com/okta/okta-developer-docs/blob/94fece47452dddf1b1b6d3b553ee5af2cd6e1c87/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Foie-embedded-widget-use-case-sign-in-soc-idp%2Fmain%2Fjava%2Fintegrationsteps.md) | 65 | 35 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/30980780-77db-4431-a7ce-8964df3e78f5) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-case-sign-in-soc-idp/main/nodejs/integrationsteps.md](https://github.com/okta/okta-developer-docs/blob/94fece47452dddf1b1b6d3b553ee5af2cd6e1c87/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Foie-embedded-widget-use-case-sign-in-soc-idp%2Fmain%2Fnodejs%2Fintegrationsteps.md) | 62 | 42 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/67e769b5-2451-4b36-a847-4506181a2fcf) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 4 of 4 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.c335de0f-bc43-4531-aec2-04bd7d6b61be?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Review by grahamsmith-okta — APPROVED** [review]
Looks good, just added the same formatting comment in a couple of places (using articles in the list for consistency and a casing change).

One thing to note about the diagram is that the "Keep me signed in" button will be replaced with "Stay signed in" in the 2.0 release (Stay signed in is currently EA, moving to GA next week).

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-case-sign-in-soc-idp/main/nodejs/integrationsteps.md [hunk 1]
```diff
@@ -2,6 +2,16 @@
 
 If the the steps in [Configuration updates](#configuration-updates) were completed, the **Sign in with Facebook** link appears automatically on the Widget. No code change is required for the Facebook sign-in link to appear.
 
+<div class="half wireframe-border">
+
+![The Okta Sign-In Widget's sign-in form with a field for a username, next button, sign in with Facebook button and links to reset your password and sign up](/img/wireframes/widget-sign-in-form-username-only-sign-up-forgot-your-password-facebook-links.png)
```
[the remark this answers is not in the document]
Comment (danielmaharry-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-case-sign-in-soc-idp/main/nodejs/integrationsteps.md:7]: fixed

**Review by grahamsmith-okta — APPROVED** [review]
lgtm

**Pull request merged**